README.md

```diff
@@ -108,6 +168,54 @@ jobs:
         SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
 ```
 
+For C, C++ and Objective-C projects not using AutoConfig, the workflow requires additional steps to download the Build Wrapper and invoking it:
+
+```yaml
+# Trigger analysis when pushing to your main branches, and when creating a pull request.
+  push:
+    branches:
+      - main
+      - master
+      - develop
+      - 'releases/**'
+  pull_request:
+      types: [opened, synchronize, reopened]
+
+name: Main Workflow
+jobs:
+  sonarqube:
+    runs-on: ubuntu-latest
+    env:
+      BUILD_WRAPPER_OUT_DIR: build_wrapper_output_directory # Directory where build-wrapper output will be placed
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        # Disabling shallow clone is recommended for improving relevancy of reporting
+        fetch-depth: 0
+    - name: Install Build Wrapper
+      uses: sonarsource/sonarqube-scan-action/install-build-wrapper@<action version>
+      env:
+        SONAR_HOST_URL: ${{ secrets.SONAR_HOST_URL }}
```

Comment:
Since we are in the `Cloud` section, what about dropping this (assuming https://sonarcloud.io is the default)?

Reply:
Yes, I agree it should be dropped for SonarQube Cloud.

I realize now that `install-build-wrapper`, unlike the main action, assumes no default for `SONAR_HOST_URL`.
I am going to add such default and test the action with SQC examples from CFamily.

Reply:
A test revealed that in `install-build-wrapper` and `deprecated-c-cpp` sub-actions nothing assumed that lack of `SONAR_HOST_URL` would mean `https://sonarcloud.io`.
 
I had to add [this](https://github.com/SonarSource/sonarqube-scan-action/pull/161/commits/fe5b156c9091460353d3f56e0ddcb5bf54a2fbb8) step to both `install-build-wrapper` and `deprecated-c-cpp` sub-actions.

It now works, you can see a successful test [here](https://github.com/antonioaversa/windows-msbuild-opencppcoverage-gh-actions-sc/actions/runs/12294180509/job/34309535901#step:6:5).

I have tested all projects in https://github.com/sonarsource-cfamily-examples?q=gh-actions-sc&type=all&language=&sort= and added the result to the PR description.